scalajslib/src/mill/scalajslib/ScalaJSModule.scala

```diff
@@ -293,6 +296,9 @@ trait ScalaJSModule extends scalalib.ScalaModule { outer =>
   /** Whether to emit a source map. */
   def scalaJSSourceMap: T[Boolean] = Task { true }
 
+  /** Whether to emit WASM. As of Nov 2024 scala JS wasm support is experimental */
+  def scalaJSEmitWasm: T[Boolean] = Task { false }
```

Comment:
I would stick to the same names used by the official Scala.js sbt plugin.
In this case since the method is `withExperimentalUseWebAssembly` then I would use:
```scala
def scalaJSExperimentalUseWebAssembly: T[Boolean] = Task { false }
```

Reply:
Then you could rename the fields and parameters accordingly in the private methods and files.

Reply:
I'm happy to do this - the caveat is that if wasm were to become non-experimental, it would break compatibility to declare it as such?

Reply:
We will follow the same process as Scala.js, deprecate the field and tell people to use the new one.
We've done this already with [useECMAScript2015](https://github.com/com-lihaoyi/mill/pull/1671/files#diff-205bc876ab2361a662d87fc8432d83a29d06f56b88de596df28dd03ae5ddd7a8R200)